src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryWorker.cs

```diff
@@ -204,6 +189,54 @@ public void Validate([NotNull] HttpContext httpContext, string cookieToken, stri
                 deserializedFormToken);
         }
 
+
+        /// <summary>
+        /// Generates an anti-forgery cookie token and header for this request.
```

Comment:
I suggest drop the Generates as it only generates if it cant find one or the existing token in not valid.